README.md

```diff
@@ -82,7 +82,7 @@ $ pip install nle
 NOTE: If you want to extend / develop NLE, please install the package as follows:
 
 ``` bash
-$ git clone git@github.com:facebookresearch/nle
+$ git clone https://github.com/facebookresearch/nle --recursive
```

Comment:
Do we have submodules right now?

Reply:
Yes, in the earlier PR we added pybind11 in `third_party/pybind11`: https://github.com/facebookresearch/nle/tree/master/third_party